manager/engine.go

```diff
@@ -478,6 +478,34 @@ func (m *VolumeManager) ListAllBackupsSorted() ([]*longhorn.Backup, error) {
 	return backups, nil
 }
 
+func (m *VolumeManager) ListBackupsForVolumeName(volumeName string) (map[string]*longhorn.Backup, error) {
+	v, err := m.ds.GetVolumeRO(volumeName)
+	if err != nil && !apierrors.IsNotFound(err) {
+		return nil, err
+	}
+	backupTargetName := ""
+	if v != nil {
+		backupTargetName = v.Spec.BackupTargetName
+	}
+	return m.ds.ListBackupsWithVolumeNameRO(volumeName, backupTargetName)
```

Comment:
What are the listed backups for an empty backupTargetName input?

Reply:
It will only select the backups with the label `backup-volume` which is equal to the parameter `volumeName` and ignore the label `backup-target`.